### ISSUE TITLE
SIP RTP packages to jitsi / docker host use container ip

### ISSUE THREAD

**Issue by ahoeg** [jitsi/docker-jitsi-meet#1033]
I don't know why, but with latest version of jitsi jigasi container communicate the container internal ip address from docker internal network. So requests from voip server don't use the dockre host ip and instead use the docker internal ip like 172.20.0.4.

Something changed here? Is it a bug or could it be configured?
Looks like jigasi add the container ip to the sip package instead of the docker host ip content?

DOCKER_HOST_ADDRESS is set to docker host ip.

**Comment by damencho** [comment]
This was always the case. Jigasi is not doing any NAT traversal or there is no setting for setting that address. It was always the responsibility of the sip service to do latching. To send packets from where packets are sent.

**Comment by damencho** [comment]
I'm talking about Jigasi in general not only in the docker context.

**Comment by ahoeg** [comment]
But something changed to the older release used. 

I know that containers use the container ip address and the docker host masquerade to the docker host ip address. That's fine!
Tcpdump shows me the docker host ip and the voip server ip address. 

But rtp packages initiated from voip server have the container internal ip address as destination! So it looks like the packages transport the container ip? Haven't seen that problem with older versions.

**Comment by nicedevil007** [comment]
I found the "location" of the docker container IP adress inside the file: ```sip-communicator.properties```
This file is located inside the volume of the videobridge (jvb in our case).

There is a line that looks like this:

```org.ice4j.ice.harvest.NAT_HARVESTER_LOCAL_ADDRESS=172.20.0.5```

I already tryed to change this line to something that is using the hosts IP adress. But this line is not used if the container is already running and if I stop the container, change my settings, start it again, the line got overwritten all the time.
